src/ra_server_proc.erl

```diff
@@ -1503,13 +1505,7 @@ follower_leader_change(Old, #state{pending_commands = Pending,
 aten_register(Node) ->
     case node() of
         Node -> ok;
-        _ ->
-            case aten:register(Node) of
-                ignore ->
-                    ok;
-                Res ->
-                    Res
-            end
+        _ -> aten:register(Node)
```

Comment:
This is not a direct equivalent (maybe this does not matter). In the original version a value of `ignore` returned by Aten would be returned as `ok` by Ra. Now that's not the case, is there a reason for that?

Reply:
Dialyze mostly! `aten:register` returns only `ok`.

Reply:
I think you have an old aten version, see https://github.com/rabbitmq/aten/blob/main/src/aten.erl#L18